payload-schemas/schemas/common/organization.schema.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
This seems to not contain all the fields.

You can cross-reference https://api.github.com/orgs/Octocoders and make sure the `company`, `blog`, `email`, and `twitter_username` are `null | string`